vlib/v/tests/c_structs/cstruct.h

```diff
@@ -9,4 +9,19 @@ typedef struct Test2 {
 
 typedef struct Test1 {
     Test2 a;
-} Test1;
\ No newline at end of file
+} Test1;
+
+/////
+
+typedef struct Foo {
+    int a;
+} Foo;
+
+typedef struct Bar {
+    int a;
+} Bar;
+
+typedef union Baz {
+    Foo foo;
+    Bar bar;
+} Baz;
```

Comment:
This is not needed. The `main.Baz` type on the V side, has no relation at all to this C `Baz`, and imho it is confusing to have that union here.

Reply:
Imho remove those lines.

Reply:
OH, I forgot to remove this one.